### ISSUE TITLE
Stop testing with osx1014 (v17+, master)?

### ISSUE THREAD

**Issue by targos** [nodejs/build#2769]
It started to fail with V8 9.5:
https://github.com/nodejs/node/pull/40178
https://ci.nodejs.org/job/node-test-commit-osx/42171/nodes=osx1014/console

The build output suggests that Xcode 10 is used on those hosts, but Node.js requirements are now Xcode >=11.

**Comment by AshCripps** [comment]
makes sense I had planned to add some intel big sur machines so I can just replace the 10.14 machines with them if there are no objections @nodejs/build?

**Comment by mhdawson** [comment]
https://github.com/nodejs/node/blob/master/BUILDING.md#platform-list does say that we support 10.13 and above so we might want to have some 10.13 with the newer xcode or bump up the minimum macOS level for 17.x ?

**Comment by targos** [comment]
> we might want to have some 10.13 with the newer xcode

Xcode 11 doesn't support 10.13. It looks like some versions of it support 10.14: https://en.wikipedia.org/wiki/Xcode#11.x_series

**Comment by AshCripps** [comment]
10.13 is our compilation target - it should be that our binaries will work on it but we dont compile on it.
